Gems/ScriptCanvas/Code/Asset/EditorAssetSystemComponent.cpp

```diff
@@ -31,6 +31,8 @@ AZ_POP_DISABLE_WARNING
 
 namespace ScriptCanvasEditor
 {
+    using namespace ScriptCanvas;
+
```

Comment:
Using namespace is banned at [global or namespace scope](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#namespaces).

This causes so many issues with name clashes in the past: https://github.com/o3de/o3de/pull/7184
Remember it is not just names from the ScriptCanvas libraries that are being introduced here.

Any dependency of ScriptCanvas like AzCore, AzFramework or a WinAPI name could now start causing clashing issues with ScriptCanvas names.